File: UEE-Pizza-hut-SL-main/backend/modals/addons.js
const mongoose = require('mongoose');
const  Schema = mongoose.Schema;

const addons = new Schema({

    name: {
        type: String,
        required : true
    },
    imageUrl:{
        type:String,
        required: true
    },
    price:{
        type: Number, 
        required: true, 
    }
})

const addon = mongoose.model("addons", addons);
module.exports = addon;

File: UEE-Pizza-hut-SL-main/lib/model/Addons.dart
import 'package:flutter/foundation.dart';

class Addons {
  late final String id;
  late final String name;
  late final int price;
  late final String imageUrl;

  Addons(
      {required this.id,
      required this.name,
      required this.price,
      required this.imageUrl});

  factory Addons.fromJson(Map<String, dynamic> json) {
    return Addons(
        id: json['_id'],
        name: json['name'],
        imageUrl: json['imageUrl'],
        price: json['price']);
  }
}
